protocol-testing/src/test_runner.rs

```
@@ -828,7 +828,7 @@ impl TestRunner {
             .chain(
                 initialized_accounts
                     .iter()
-                    .map(|account| Bytes::from(account.as_bytes())),
+                    .filter_map(|account| Bytes::from_str(account).ok()),
```

I'll be picky here but for a better dev experience we should cast this into `Bytes` way earlier, like at the beginning of the test, even before we sync. Then we can panic and error if there is something misformatted. 

Because here we would just hide the error and this would confuse someone for sure if it happens

---

This is already handled in a separate PR #343